test/fixtures/check-direct-import-transitive-dependency.txt

```diff
@@ -8,4 +8,4 @@ To fix this error add the following packages to the list of dependencies in your
 - maybe
 - prelude
 You may add these dependencies by running the following command:
-spago install maybe prelude[0m
+spago install maybe prelude[0m
```

Comment:
Don't know what non-printable characters changed here. Guess the lesson moving forward is to not rely on web-UI for conflict resolution.